packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -46,7 +46,8 @@ import {
 } from "@fluidframework/container-utils";
 import { IConnectionManagerFactoryArgs, IConnectionManager } from "./contracts";
 import { DeltaQueue } from "./deltaQueue";
-import { OnlyValidTermValue } from "./protocol";
+// "term" was an experimental feature that is being removed.  The only safe value to use is 1.
+export const OnlyValidTermValue = 1 as const;
```

Comment:
is there a bug to clean these? Or is this here forever?

Reply:
I'll follow-up as I'm not sure who is working on deprecating this feature. The reason I did not remove it completely is due to the additional changes I'd have to make to update dependencies around it.